woocommerce-payments.php

```diff
@@ -21,6 +21,7 @@
 define( 'WCPAY_PLUGIN_FILE', __FILE__ );
 define( 'WCPAY_ABSPATH', __DIR__ . '/' );
 define( 'WCPAY_MIN_WC_ADMIN_VERSION', '0.23.2' );
+defined( 'JETPACK__WPCOM_JSON_API_BASE' ) || define( 'JETPACK__WPCOM_JSON_API_BASE', 'https://public-api.wordpress.com' );
```

Comment:
Not sure if it's needed - see https://github.com/Automattic/jetpack/blob/fd64fa48f39b91e147ac7fd1bafba7c294f96aa8/projects/plugins/jetpack/jetpack.php#L64 & https://github.com/Automattic/jetpack/pull/19688 (wasn't working for me in dev mode). Maybe it's because we're using an older version of the jetpack composer package, I haven't investigated too deeply